spec/services/iaa_reporting_helper_spec.rb

```diff
@@ -145,4 +167,43 @@ def build_integration(issuer:, partner_account:)
       end
     end
   end
+
+  describe '#partneraccounts' do
```

Comment:
this should match the name of the method it's testing
```suggestion
  describe '#partner_accounts' do
```